src/Neo.VM/ExecutionEngine.cs

```diff
@@ -166,7 +166,7 @@ protected internal void ExecuteNext()
         /// Loads the specified context into the invocation stack.
         /// </summary>
         /// <param name="context">The context to load.</param>
-        internal virtual void LoadContext(ExecutionContext context)
+        protected internal virtual void LoadContext(ExecutionContext context)
```

Comment:
If they both need to be `internal` then make it `public`. When a `3rd-party` does `public class MyApplicationEngine : ApplicationEngine` it needs to work with `inheritances`. For example `neo-express` uses this and I can't `override` the method so I can update for latest release of `neo`.

Reply:
it can be extended by neo-express with protected internal